### PR TITLE
Issue#7,22 IMPLEMENTATION - Create Screen - Favorites and Create Data…

### PR DESCRIPTION
**Pull request by heatsink**
Created a Favorites Page screen using existing implementation.

Closing Issue #22 (Data Storage Mechanism) to be fully implemented later in Sprint 2.
https://github.com/mobileappdevhm/dev-team-1-cie-app-in-flutter/issues/7
https://github.com/mobileappdevhm/dev-team-1-cie-app-in-flutter/issues/22

### CODE REVIEW

**Comment by AndiAndreas** [comment]
00:12 +19 -1: loading /home/travis/build/mobileappdevhm/dev-team-1-cie-app-in-flutter/test/utils/fileStore_test.dart [E]                                                                               
  Failed to load "/home/travis/build/mobileappdevhm/dev-team-1-cie-app-in-flutter/test/utils/fileStore_test.dart":

File: .gitignore [hunk 1]
```diff
@@ -6,11 +6,13 @@
...
 packages
 .flutter-plugins
 
 #Ignore IOS specific files
 ios/Podfile
 ios/Podfile.lock
 ios/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
+ios/Runner.xcodeproj/project.pbxproj
+ios/.generated/
+pubspec.lock
```
Comment (RedTo) [inline .gitignore]: I have read somewhere, that pubspec.lock should stay in version control... But I'm not quite sure about it...

File: lib/views/tabs.dart [hunk 2]
```diff
@@ -77,6 +79,7 @@ class TabsPageState extends State<TabsPage> {
   void _onPageChanged(int tab) {
     setState(() {
       this._tab = tab;
+      this._shouldFilter = !this._shouldFilter;
```
Comment (RedTo) [inline lib/views/tabs.dart]: Is this right on every page change?

**Comment by heatsink** [comment]
Issue#7, 22 IMPLEMENTATION CONTINUED - Removed pubspec.lock from gitignore, revised courseList filter logic, removing empty test file that made Travis angry

File: lib/views/tabs.dart (hunk 2 above)
Comment (heatsink) [reply lib/views/tabs.dart]: Woops, good catch.

**Pull request merged**